PowerPointLabs/PowerPointLabs/FrameMotionAnimation.cs

```diff
@@ -92,10 +103,10 @@ public static void AddZoomToAreaPanFrameMotionAnimation(PowerPointSlide animatio
             float incrementLeft = (finalX - initialX) / numFrames;
             float incrementTop = (finalY - initialY) / numFrames;
 
-            AddFrameAnimationEffects(animationSlide, initialShape, incrementLeft, incrementTop, incrementWidth, incrementHeight, 0.0f, 0.0f, duration, numFrames);
+            AddFrameAnimationEffects(animationSlide, initialShape, incrementLeft, incrementTop, incrementWidth, incrementHeight, false, false, 0.0f, 0.0f, duration, numFrames);

```

Comment:
add a prefix of param name to indicate the meaning of the boolean value,
e.g. if the param name is isFlipHorizontally, then `isFlipHorizontally: false`